plugins/chat/app/services/chat/create_message.rb

```diff
@@ -222,6 +223,10 @@ def process(channel:, message_instance:, contract:, thread:)
       end
     end
 
+    def create_excerpt(message_instance:)
+      message_instance.excerpt = WordWatcher.censor(message_instance.raw_excerpt)
```

Comment:
can this raise an exception in any circumstance?

Reply:
I don't think so, unless I am missing something. It will allow a null value in the column when there is no message (ie. if PrettyText.excerpt returns a blank message).